### PR TITLE
Add blog post about May 26th plugin update

### PR DESCRIPTION
**Pull request by azigler**
This PR adds a blog post about this week's `mattermost-plugin-ai` update summarized here:  https://community.mattermost.com/core/pl/ih9qbrx6ipfj9r6pcgbmdmna4w

### CODE REVIEW

File: content/posts/may-26-plugin-update.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+🤖 **Renamed as AI Assistant**: We have renamed the LLM Bot to AI Assistant for better clarity. Now, you can easily call upon the AI Assistant by mentioning `@ai` instead of `@llmbot`.
+
+🎨 **Updated Profile Icon**: The AI Assistant now sports a brand new profile icon, giving it a fresh and distinctive look.
+
+👍👎 **Enhanced Feedback Reactions**: The thumbs-up and thumbs-down reactions have received a visual improvement and send feedback to the AI Assistant.
+
+🔠 **Font Reverted to Regular**: We have reverted from using a monospaced font to a regular font. However, don't worry! We are working on introducing additional controls to differentiate the AI Assistant from regular users so there is no confusion. 
+
+AI open source technologies continue to evolve, empowering us to create intelligent and collaborative tools that augment human capabilities. Let's embrace the potential of AI and open source to shape a future where intelligent assistants empower us to achieve greater productivity and success.
```
Comment (jasonblais) [inline content/posts/may-26-plugin-update.md]: @azigler what are your thoughts on adding a CTA, e.g. to the GitHub repo or the community channel?

Comment (azigler) [reply content/posts/may-26-plugin-update.md]: @jasonblais Great idea! I updated the blog post to use @asaadmahmood's new metadata format, added CTAs for the channel and forums, and changed the repo name to be a link to GitHub. I updated the date to today and attributed it to @crspeller.

**Review by jasonblais — APPROVED** [review]

**Comment by azigler** [comment]
@asaadmahmood Can you merge when you have a chance?

**Comment by asaadmahmood** [comment]
@jasonblais @azigler We need to run `hugo -d docs` before pushing any changes.

**Comment by azigler** [comment]
> @jasonblais @azigler We need to run `hugo -d docs` before pushing any changes.

Gotcha. I went ahead and did that now. @jasonblais should we consider setting up some automations for this process?

**Comment by asaadmahmood** [comment]
I have pushed this separately to avoid the conflicts.

**Comment by azigler** [comment]
@asaadmahmood Looks good but seems like the Markdown for `inline code` is not being rendered in the title:

https://ai.mattermost.com/posts/2023-05-30-plugin-update/

The title here (and on the homepage) is "May 26th update to \`mattermost-plugin-ai\`"

**Comment by asaadmahmood** [comment]
@azigler I'm not sure if the title allows the markdown to render.

**Comment by azigler** [comment]
@asaadmahmood Gotcha. If so, can you remove them from that title when you get a chance?

**Comment by asaadmahmood** [comment]
@azigler Done.

**Pull request closed without merging**